soap/src/java/com/zimbra/soap/mail/type/EditheaderTest.java

```diff
@@ -0,0 +1,254 @@
+
+    /**
+     * @return the count
+     */
+    public Boolean getCount() {
+        return count;
+    }
+
+    /**
+     * @param count the count to set
+     */
+    public void setCount(Boolean count) {
+        this.count = count;
+    }
+
+    /**
+     * @return the value
+     */
+    public Boolean getValue() {
+        return value;
+    }
+
+    /**
+     * @param value the value to set
+     */
+    public void setValue(Boolean value) {
+        this.value = value;
+    }
+
+    /**
+     * @return the relationalComparator
+     */
+    public String getRelationalComparator() {
+        return relationalComparator;
+    }
+
+    /**
+     * @param relationalComparator the relationalComparator to set
+     */
+    public void setRelationalComparator(String relationalComparator) {
+        this.relationalComparator = relationalComparator;
+    }
+
+    /**
+     * @return the comparator
+     */
+    public String getComparator() {
+        return comparator;
+    }
+
+    /**
+     * @param comparator the comparator to set
+     */
+    public void setComparator(String comparator) {
+        this.comparator = comparator;
+    }
+
+    /**
+     * @return the headerName
+     */
+    public String getHeaderName() {
+        return headerName;
+    }
+
+    /**
+     * @param headerName the headerName to set
+     */
+    public void setHeaderName(String headerName) {
+        this.headerName = headerName;
+    }
+
+    /**
+     * @return the headerValue
+     */
+    public List<String> getHeaderValue() {
+        return headerValue;
+    }
+
+    /**
+     * @param headerValue the headerValue to set
+     */
+    public void setHeaderValue(List<String> headerValue) {
+        this.headerValue = headerValue;
+    }
+
+    /**
+     * @param headerValue the headerValue to add
+     */
+    public void addHeaderValue(String headerValue) {
+        if (this.headerValue == null) {
+            this.headerValue = new ArrayList<String>();
+        }
+        this.headerValue.add(headerValue);
+    }
+
+    /**
+     * @param headerValue the headerValues to add
+     */
+    public void addAllHeaderValues(List<String> headerValue) {
+        if (this.headerValue == null) {
+            this.headerValue = new ArrayList<String>();
+        }
+        this.headerValue.addAll(headerValue);
+    }
+
+    @Override
+    public String toString() {
+        return Objects.toStringHelper(this)
+            .add("matchType", matchType)
+            .add("count", count)
+            .add("value", value)
+            .add("relationalComparator", relationalComparator)
+            .add("comparator", comparator)
+            .add("headerName", headerName)
+            .add("headerValue", headerValue)
+            .toString();
+    }
+
+    public void validateEditheaderTest() throws ServiceException {
+        if (StringUtil.isNullOrEmpty(headerName)) {
+            throw ServiceException.PARSE_ERROR("Invalid EditheaderTest: Missing headerName", null);
```

Comment:
sieve script is being logged at debug level before processing, so no need to log it again.